admin_site/system/models.py

```diff
@@ -1215,3 +1225,20 @@ def __str__(self):
 
     def __unicode__(self):
         return "{0}: {1}".format(self.problem.name, self.id)
+
+
+class ImageVersion(models.Model):
+    img_vers = models.CharField(unique=True, max_length=7)
+    rel_date = models.DateField()
+    os = models.CharField(max_length=30)
+    rel_notes = models.TextField(max_length=350)
+    img_path = models.CharField(max_length=200)
+
+    def __str__(self):
+        return "| {0} | {1} | {2} | {3} | {4} |".format(
+            self.img_vers,
+            self.rel_date,
+            self.os,
+            self.rel_notes,
+            self.img_path
+            )
```

Comment:
A question of style: I think the closing paranthesis should match the line on which it starts, i.e.:

```
    return "sdasdas".format(
        ...
    )
```
rather than as you do above.

Reply:
Jeg er nede med begge approaches, så retter det gerne til. I henhold til pep8, så mener jeg kun det er konvention, når der er tale om deklaration af en function. Hvorimod return statements, lister osv., da er begge approaches gyldige ift. pep8.